frontend/src/container/GridGraphLayout/Graph/FullView/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import { LabelContainer } from './styles';
+import { getAbbreviatedLabel } from './utils';
+
+function Label({
+	labelClickedHandler,
+	labelIndex,
+	label,
+}: LabelProps): JSX.Element {
+	return (
+		<LabelContainer
+			type="button"
+			onClick={(): void => labelClickedHandler(labelIndex)}
```

Comment:
props should have one to one mapping inline functions is not recommended can you please update the function to closure

Reply:
Okay.